src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/switchOrganization.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import DropdownMenu from 'app/components/dropdownMenu';
+import {IconAdd} from 'app/icons/iconAdd';
+import withOrganizations from 'app/utils/withOrganizations';
+import SidebarOrgSummary from 'app/components/sidebar/sidebarOrgSummary';
+import SidebarMenuItem from 'app/components/sidebar/sidebarMenuItem';
+import SidebarDropdownMenu from 'app/components/sidebar/sidebarDropdownMenu.styled';
+import {OrganizationSummary} from 'app/types';
+
+import Divider from './divider.styled';
+
+type Props = {
+  organizations: OrganizationSummary[];
+  canCreateOrganization: boolean;
+};
+/**
+ * Switch Organization Menu Label + Sub Menu
+ */
+const SwitchOrganization = ({organizations, canCreateOrganization}: Props) => (
+  <DropdownMenu isNestedDropdown>
+    {({isOpen, getMenuProps, getActorProps}) => (
+      <React.Fragment>
+        <SwitchOrganizationMenuActor
+          data-test-id="sidebar-switch-org"
+          {...getActorProps({})}
+          onClick={e => {
+            // This overwrites `DropdownMenu.getActorProps.onClick` which normally handles clicks on actor
+            // to toggle visibility of menu. Instead, do nothing because it is nested and we only want it
+            // to appear when hovered on. Will also stop menu from closing when clicked on (which seems to be common
+            // behavior);
+
+            // Stop propagation so that dropdown menu doesn't close here
+            e.stopPropagation();
+          }}
+        >
+          {t('Switch organization')}
+
+          <SubMenuCaret>
+            <i className="icon-arrow-right" />
+          </SubMenuCaret>
+        </SwitchOrganizationMenuActor>
+
+        {isOpen && (
+          <SwitchOrganizationMenu
+            data-test-id="sidebar-switch-org-menu"
+            {...getMenuProps({})}
+          >
+            <OrganizationList>
+              {organizations.map(organization => {
+                const url = `/organizations/${organization.slug}/`;
+
+                return (
+                  <SidebarMenuItem key={organization.slug} to={url}>
+                    <SidebarOrgSummary organization={organization} />
+                  </SidebarMenuItem>
+                );
+              })}
+            </OrganizationList>
+            {organizations && !!organizations.length && canCreateOrganization && (
+              <Divider css={{marginTop: 0}} />
```

Comment:
isn't it better to style the component using styled(Divider)....?

Reply:
Yeah probably, but this is how it was, so going to leave it.